src/test/kotlin/com/zepben/evolve/cim/iec61970/base/core/NameTypeTest.kt

```diff
@@ -92,7 +97,7 @@ internal class NameTypeTest {
     }
 
     @Test
-    fun removesNames() {
+    fun `removesNames remove names from nameType and associated identifiedObject`() {
```

Comment:
todo: Test name indicates this should be checking that the name is removed from the `IdentifiedObject`, but there is no check of this happening. This also applies to the `removeName` version below

Reply:
Done